README.md

```diff
@@ -2,23 +2,17 @@
 
 - **Version**: 1.0.0 
 - **Certified**: No 
-- **Publisher**: Community 
+- **Publisher**: Fortinet 
 - **Compatible Version**: 7.3.1 and later 
 
-<table>
-    <tr>
-        <th>Note</th>
-        <td>Configure the <a href="https://docs.fortinet.com/fortisoar/connectors/remotefsr">Remote FortiSOAR</a> connector for this solution pack to work seamlessly. The configuration wizard appears right after the installation of this solution pack.</td>
-    </tr>
-</table>
-
 # Overview 
-The **Global Visibility** solution pack offers a user-friendly dashboard for accessing data from multiple sites. To enable this, users must set up the Remote FortiSOAR connector for each site, facilitating data retrieval for the dashboard widgets.
-
-This streamlined configuration process ensures seamless access to crucial information across various locations, enhancing overall visibility and decision-making capabilities.
+The **Global Visibility** solution pack offers a user-friendly dashboard for visualize and accessing data across multiple FortiSOAR instances. 
+This streamlined configuration process ensures seamless access to crucial information across various locations, enhancing overall visibility and decision-making capabilities for any SOC.
 
 The **Global Visibility** solution pack is simplifying the task of aggregating site-specific data, providing a comprehensive and centralized view for users, and ultimately empowering the users to make informed choices efficiently.
 
+![global_visibility_dashboard](./docs/res/global-visibility-dashboard-01.png)
```

Comment:
Please change "global_visibility_dashboard" to "Global Visibility Dashboard", since this is an alt text